compiler/rustc_trait_selection/src/solve/normalizes_to/mod.rs

```diff
@@ -889,6 +882,43 @@ impl<'tcx> assembly::GoalKind<'tcx> for NormalizesTo<'tcx> {
     }
 }
 
+fn translate_args<'tcx>(
+    ecx: &mut EvalCtxt<'_, InferCtxt<'tcx>>,
```

Comment:
why is that not a method on `EvalCtxt`? :sweat_smile:

Reply:
🤷 `fetch_eligible_assoc_item_def` isn't one either

Reply:
i feel like both should be, r=me with or without that change